incubator/group/types.go

```diff
@@ -0,0 +1,56 @@
+package group
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/modules/incubator/orm"
+
+	"time"
+)
+
+// TODO: what makes sense here?
```

Comment:
This should be a `param` that can be changed.

Reply:
I have implemented it into parameter subspace, now.